render/wgpu/src/shaders.rs

```diff
@@ -0,0 +1,51 @@
+#[derive(Debug)]
+pub struct Shaders {
+    pub color_shader: wgpu::ShaderModule,
+    pub bitmap_shader: wgpu::ShaderModule,
+    pub gradient_shader: wgpu::ShaderModule,
+    pub copy_srgb_shader: wgpu::ShaderModule,
+}
+
+impl Shaders {
+    pub fn new(device: &wgpu::Device) -> Self {
+        let color_shader = create_shader(device, "color", include_str!("../shaders/color.wgsl"));
+        let bitmap_shader = create_shader(device, "bitmap", include_str!("../shaders/bitmap.wgsl"));
+        let gradient_shader = if device.limits().max_storage_buffers_per_shader_stage > 0 {
```

Comment:
Is this intentionally shadowed by `let gradient_shader = create_shader(device, "gradient", gradient_shader);`?